test/service.go

```diff
@@ -233,3 +266,15 @@ func RouteHasHost(r *routev1.Route, err error) (bool, error) {
 }
 
 func int32Ptr(i int32) *int32 { return &i }
+
+func UpdateGlobalProxy(ctx *Context, value string) error {
```

Comment:
So this file is called `service.go`. Can we create a new file `proxy.go` that then gets the new helpers?